src/unix.rs

```diff
@@ -6,27 +6,24 @@
 use std::env;
 use std::ffi::OsStr;
 
-use lazy_static::lazy_static;
 use nix::unistd::{sysconf, SysconfVar};
+use once_cell::sync::Lazy;
 
 use crate::constants::POINTER_SIZE_CONSERVATIVE;
 
 const UPPER_BOUND_COMMAND_LINE_LENGTH: i64 = 16 * 1024 * 1024;
 
-lazy_static! {
-    static ref PAGE_SIZE: i64 = {
-        sysconf(SysconfVar::PAGE_SIZE)
-            .ok()
-            .flatten()
-            .map(|page_size| page_size as i64)
```

Comment:
The cast is still needed for 32bit platforms. We should probably add a comment why it's there.

Reply:
Done, sorry, should have asked before removing